pytest_docker_tools/factories/container.py

```diff
@@ -8,16 +8,49 @@
 def container(request, docker_client, wrapper_class, **kwargs):
     """ Docker container: image={image} """
 
+    wrapper_class = wrapper_class or Container
+
+    if request.config.option.reuse_containers:
+        if "name" in kwargs.keys():
+            name = kwargs["name"]
+            current_containers = docker_client.containers.list(ignore_removed=True)
+            for cont in current_containers:
+                if cont.name == name:
+                    if (
+                        "Pytest-Docker-Tools.Reusable_Container"
+                        in cont.attrs["Config"]["Labels"]
+                        and cont.attrs["Config"]["Labels"][
+                            "Pytest-Docker-Tools.Reusable_Container"
+                        ]
+                        == "True"
+                    ):
+                        return wrapper_class(cont)
+        else:
+            raise RuntimeError(
+                "Error: Tried to use '--reuse_containers' command line argument without "
+                "setting 'name' attribute on container"
+            )
+
     timeout = kwargs.pop("timeout", 30)
 
     kwargs.update({"detach": True})
+    kwargs.update(
+        {
+            "labels": {
+                "Container_Creator": "Pytest-Docker-Tools",
```

Comment:
Can we use `pytest-docker-tools` rather than `Pytest-Docker-Tools` everywhere?

Reply:
And i know it's really a pointless thing to care about but could I have `container-creator` and `pytest-docker-tools.reusable-container`?

Reply:
sure